src/getCoinCombination.test.js

```diff
@@ -7,4 +7,27 @@ describe('getCoinCombination', () => {
     expect(getCoinCombination)
       .toBeInstanceOf(Function);
   });
+
+  it('should return an array', () => {
+    expect(getCoinCombination())
```

Comment:
The test case calls `getCoinCombination()` without any arguments. Ensure that the function can handle this call or provide a valid input, as the function is expected to take a non-negative integer as input according to the task description.